scripts/check_samples.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import glob
+import sys
+
+def check_template_sample(template_file_name):
+
+    # retrieve the template kind
+    with open(template_file_name, 'r') as template_file:
+        
+        template_kind = re.findall('kind:\s+(\w+)', template_file.read())[1]
```

Comment:
Let's parse this via yaml rather than regex.

Reply:
done